.gitignore

```diff
@@ -10,3 +10,4 @@ nbproject
 spec/dummy
 pkg
 Gemfile.lock
+vendor/
```

Comment:
This is project specific, I don't think. Should be either in your global gitignore, or your local excludes.